source/upgrade/important-upgrade-notes.rst

```diff
@@ -21,6 +21,10 @@ Important Upgrade Notes
 +----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
 +====================================================+==================================================================================================================================================================+
+| v6.2                                               | Channel results in the channel menu will include channels from other teams as well as private channels. Customers using Bleve or Elasticsearch for               |
```

Comment:
I don't think the cross team autocomplete was done yet? My understanding is that the front end is still outstanding? cc// @agnivade

Reply:
Is a reindex still required even for just private channel autocomplete?

Reply:
My apologies, I misread it quickly, yes cross team autocomplete isn't done. Only private channel autocomplete is.

Yes, reindex is still required even for private channel autocomplete.

Reply:
I edited the text.

Reply:
Hey @amyblais - This is specifically for the channel autocomplete, not the channel switcher. Channel switcher always showed private channels.